index.html

```diff
@@ -14,18 +13,69 @@
     <meta property="og:title" content="Flipkart Skeleton UI">
     <meta property="og:description" content="Flipkart Skeleton UI - This Project is a single-page skeleton UI clone of flipkart's main page.">
     <meta property="og:type" content="website">
-    <meta property="og:image" content="/asset/image/flipkart_cover_page.jpg">
+    <meta property="og:image" content="/assets/image/flipkart_cover_page.jpg">
 
     <!-- Twitter meta tags -->
     <meta name="twitter:card" content="summary_large_image">
     <meta name="twitter:title" content="Flipkart Skeleton UI">
     <meta name="twitter:description" content="Flipkart Skeleton UI - This Project is a single-page skeleton UI clone of flipkart's main page.">
-    <meta name="twitter:image" content="/asset/image/flipkart_cover_page.jpg"> 
+    <meta name="twitter:image" content="/assets/image/flipkart_cover_page.jpg"> 
+
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.3/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-QWTKZyjpPEjISv5WaRU9OFeRpok6YctnYmDr5pNlyT2bRjXh0JMhjY6hW+ALEwIH" crossorigin="anonymous">
+    <script defer src="https://cdn.jsdelivr.net/npm/bootstrap@5.3.3/dist/js/bootstrap.bundle.min.js" integrity="sha384-YvpcrYf0tY3lHB60NNkmXc5s9fDVZLESaAA55NDzOxhy9GkcIdslK1eN7N6jIeHz" crossorigin="anonymous"></script>
 
-    <link rel="icon" href="/asset/icon/flipkart.svg">
-    <link rel="stylesheet" href="/styles/index.css">
+    <link rel="icon" href="/assets/icon/flipkart.svg">
+    <link rel="stylesheet" href="/styles/header.css">
 </head>
 <body>
-    
+    <header class="container-fluid bg-primary">
+        <div class="container h-100 d-flex justify-content-start align-items-center gap-3">
+            <a href="#" class="h-100 p-2" role="button">
```

Comment:
I think we don't required `role="button"` here. Any reason why we kept it here?
```suggestion
            <a href="#" class="h-100 p-2">
```

Reply:
Removed `role` from `a` tag

https://github.com/krushna-simform/flipkart-skeleton-ui/pull/4/commits/bd2b1a18134414dc82579fbc1a336c57315f89fd